br-webui/server1.js

```diff
@@ -0,0 +1,191 @@
+/*Recovery Server for Companion*/
```

Comment:
How this file is being executed ? There is no modification in the screen script.

Reply:
right now i havent added any modification in companion.rc. this file is executed simply by typing node server1.js. I havent added any explicit name because its just a trial version of the server. I will change the name.